src/components/Layout/Navbar.vue

```diff
@@ -11,9 +11,26 @@
         <v-btn id="nav-user" text to="/user">Account</v-btn>
         <v-btn id="nav-logout" text to="/logout">Logout</v-btn>
       </template>
-      <template v-if="!isLoggedIn">
-        <v-btn id="nav-login" text to="/login">Log in</v-btn>
-        <v-btn id="nav-create-account" ext color="primary" to="/create-account">Sign up</v-btn>
+      <v-menu bottom left v-if="($vuetify.breakpoint.width < 434)  && !isLoggedIn">
```

Comment:
I check figma design and Charlie set the breakpoint (the point where login/logout buttons merge into dots button) at 434px. I want to consult with Charlie to up the breakpoint to =<576px which cover larger number of mobile devices and 576px is also the default breakpoint of VueJS.

Reply:
But I'm happy as it is now. I also check some JS framework and their `small` breakpoints are around 460-600px